rust-project.json

```diff
@@ -0,0 +1 @@
xternal/raze__num_enum__0_5_7/src/lib.rs","edition":"2018","deps":[{"crate":113,"name":"num_enum_derive"}],"is_workspace_member":false,"cfg":["test","debug_assertions","feature=\"default\"","feature=\"std\""],"target":"x86_64-unknown-linux-gnu","env":{},"is_proc_macro":false},{"display_name":"opentitanlib","root_module":"sw/host/opentitanlib/src/lib.rs","edition":"2021","deps":[{"crate":0,"name":"mundane"},{"crate":78,"name":"anyhow"},{"crate":3,"name":"bitflags"},{"crate":4,"name":"byteorder"},{"crate":107,"name":"deser_hjson"},{"crate":108,"name":"erased_serde"},{"crate":10,"name":"hex"},{"crate":11,"name":"humantime"},{"crate":57,"name":"lazy_static"},{"crate":18,"name":"log"},{"crate":20,"name":"memoffset"},{"crate":22,"name":"mio"},{"crate":23,"name":"mio_signals"},{"crate":59,"name":"nix"},{"crate":109,"name":"num_bigint_dig"},{"crate":114,"name":"num_enum"},{"crate":25,"name":"num_traits"},{"crate":101,"name":"rand"},{"crate":68,"name":"regex"},{"crate":31,"name":"rusb"},{"crate":102,"name":"serde"},{"crate":103,"name":"serde_json"},{"crate":69,"name":"serialport"},{"crate":94,"name":"sha2"},{"crate":111,"name":"structopt"},{"crate":95,"name":"thiserror"},{"crate":112,"name":"typetag"},{"crate":96,"name":"zerocopy"},{"crate":46,"name":"safe_ftdi"},{"crate":106,"name":"opentitantool_derive"}],"is_workspace_member":true,"cfg":["test","debug_assertions"],"target":"x86_64-unknown-linux-gnu","env":{},"is_proc_macro":false},{"display_name":"opentitansession","root_module":"sw/host/opentitansession/src/main.rs","edition":"2021","deps":[{"crate":78,"name":"anyhow"},{"crate":85,"name":"directories"},{"crate":86,"name":"env_logger"},{"crate":108,"name":"erased_serde"},{"crate":10,"name":"hex"},{"crate":88,"name":"indicatif"},{"crate":18,"name":"log"},{"crate":59,"name":"nix"},{"crate":92,"name":"raw_tty"},{"crate":68,"name":"regex"},{"crate":102,"name":"serde"},{"crate":103,"name":"serde_json"},{"crate":70,"name":"shellwords"},{"crate":111,"name":"structopt"},{"crate":95,"name":"thiserror"},{"crate":115,"name":"opentitanlib"}],"is_workspace_member":true,"cfg":["test","debug_assertions"],"target":"x86_64-unknown-linux-gnu","env":{},"is_proc_macro":false},{"display_name":"opentitantool","root_module":"sw/host/opentitantool/src/main.rs","edition":"2021","deps":[{"crate":78,"name":"anyhow"},{"crate":85,"name":"directories"},{"crate":86,"name":"env_logger"},{"crate":108,"name":"erased_serde"},{"crate":10,"name":"hex"},{"crate":11,"name":"humantime"},{"crate":88,"name":"indicatif"},{"crate":18,"name":"log"},{"crate":59,"name":"nix"},{"crate":92,"name":"raw_tty"},{"crate":68,"name":"regex"},{"crate":102,"name":"serde"},{"crate":103,"name":"serde_json"},{"crate":70,"name":"shellwords"},{"crate":111,"name":"structopt"},{"crate":95,"name":"thiserror"},{"crate":106,"name":"opentitantool_derive"},{"crate":115,"name":"opentitanlib"}],"is_workspace_member":true,"cfg":["test","debug_assertions"],"target":"x86_64-unknown-linux-gnu","env":{},"is_proc_macro":false}]}
```

Comment:
I'm guessing you didn't mean to check this file in?

Reply:
Fixed.